core/find_apk.sh

```diff
@@ -8,7 +8,15 @@ getlatestapk() {
   tempapks=( )
   OLDIFS="$IFS"
   IFS="
-" # We set IFS to newline here so that spaces can survive the for loop
+" # We set IFS to newline here so that filenames containing spaces are
+  # handled correctly in the for-loops below.
+
+  # decompress apks
+  # some apks are lz compressed to workaround github file-size limits.
+  for foundapklz in $(find "$1" -iname '*.apk.lz'); do
+    lunzip $foundapklz
```

Comment:
@mfonville:  agreed.  it is better to use quotes.  change made.

@Blystad: I've changed the command to "lunzip --keep".

**re clean-up.**  This is important.  If "foo.apk.lz" is updated in a subsequent commit, then "foo.apk" won't be regenerated if that file already exists.  However, I don't think the clean-up code can go inside the script "find_apk.sh".

Ideally, the decompressed apks would reside under $OUT, and then they would be removed when "make clean" or "make installclean" is run.  But that would likely require a major refactor of "find_apk.sh".

I will think more about the clean-up, but I'd like to defer that to later PR.

If the current PR is merged, then aosp_build will start working on bullhead.